### DISCUSSION TITLE
[DONE]  .Net SDK

### DISCUSSION THREAD

**Discussion by jjhendriksen78** [featbit/featbit#219] [General]
Hi, any idea when the (C#) .NET SDK will be available?

**Comment by cosmic-flood** [comment]
Duplicate of #113 

Hi,   .NET SDK (the basic version) is expected to be available by the end of February 2023

**Reply by jjhendriksen78** [reply]
Thanks, I'll keep an eye out! Amazing library by the looks of it!

**Comment by cosmic-flood** [comment]
.NET SDK repo, work in progress

https://github.com/featbit/dotnet-server-sdk

**Comment by cosmic-flood** [comment]
> Hi, any idea when the (C#) .NET SDK will be available?

Hi @jjhendriksen78 

Version 1.0.0 of the DotNET SDK has been released.
Version 1.1.0 will be available soon.

You can visit our Documentation and SDK GitHub page for more information
https://featbit.gitbook.io/docs/getting-started/4.-connect-an-sdk/server-side-sdks/.net-sdk
https://github.com/featbit/dotnet-server-sdk

**Reply by jjhendriksen78** [reply]
Thanks very much for the heads up, I'll be sure to check it out!
